recipe/patches/1-support-zippered.patch

```diff
@@ -0,0 +1,78 @@
+From 90d0bae361b0af326b07887252a70b7d9256a9f8 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Thomas=20Pl=C3=B6chtrager?= <t.poechtrager@gmail.com>
+Date: Thu, 30 May 2019 11:35:48 +0200
+Subject: [PATCH] An attempt to get TAPI platform "zippered" working
+
+---
+ src/llvm/projects/libtapi/include/tapi/Core/Platform.h    | 2 ++
+ .../projects/libtapi/include/tapi/LinkerInterfaceFile.h   | 2 ++
+ src/llvm/projects/libtapi/lib/Core/Platform.cpp           | 8 ++++++++
+ src/llvm/projects/libtapi/lib/Core/YAML.cpp               | 3 +++
+ 4 files changed, 15 insertions(+)
+
+diff --git a/src/llvm/projects/libtapi/include/tapi/Core/Platform.h b/src/llvm/projects/libtapi/include/tapi/Core/Platform.h
+index afe19df8a..3293d48a7 100644
+--- a/src/llvm/projects/libtapi/include/tapi/Core/Platform.h
++++ b/src/llvm/projects/libtapi/include/tapi/Core/Platform.h
```

Comment:
These paths are wrong. You need to remove `src/llvm/projects/libtapi`.

Reply:
I think conda-build (or maybe `patch`) is smart enough to fix them automatically:
```
Applying patch: '/Users/christopherburr/Development/conda-forge/tapi-feedstock/recipe/patches/1-support-zippered.patch'
Trying to apply patch as-is
INFO:conda_build.source:Trying to apply patch as-is
INFO conda_build.source:apply_patch(596): Trying to apply patch as-is
patching file include/tapi/Core/Platform.h
patching file include/tapi/LinkerInterfaceFile.h
patching file lib/Core/Platform.cpp
patching file lib/Core/YAML.cpp
Applying patch: '/Users/christopherburr/Development/conda-forge/tapi-feedstock/recipe/patches/2-support-zippered.patch'
patching file tools/libtapi/LinkerInterfaceFile.cpp
```